elide-core/pom.xml

```diff
@@ -43,16 +43,15 @@
         <tag>HEAD</tag>
     </scm>
 
+    <properties>
+        <junit.version>5.4.1</junit.version>
+    </properties>
+
     <dependencies>
         <dependency>
             <groupId>com.yahoo.elide</groupId>
             <artifactId>elide-annotations</artifactId>
         </dependency>
-        <dependency>
-            <groupId>com.yahoo.elide</groupId>
-            <artifactId>elide-example-models</artifactId>
```

Comment:
The point of elide-example-models was to move the common models from core and integration tests to a central place.  This works against that.  Let's leave example models alone for now.